File: Kotlin/entradaSaida/Exercicio1.kt
package entradaSaida

fun main(){
    print("Digite a altura do aquário (cm): ")
    val altura = readln().toDouble()
    print("Digite o comprimento do aquário (cm): ")
    val comprimento = readln().toDouble()
    print("Digite a profundidade do aquário (cm): ")
    val profundidade = readln().toDouble()

    val volume = (altura * comprimento * profundidade) / 1000

    print("O volume do aquário é de: $volume l")
}

File: Ruby_/entradaSaida/Exercicio1.rb
# frozen_string_literal: true

print "Digite a altura do aquário (cm): "
altura = gets.to_f
print "Digite o comprimento do aquário (cm): "
comprimento = gets.to_f
print "Digite a profundidade do aquário (cm): "
profundidade = gets.to_f

volume = (altura * comprimento * profundidade) / 1000
puts "O volume do aquário equivale a: #{volume}l"